oggm/utils.py

```diff
@@ -1970,6 +1970,51 @@ def write_monthly_climate_file(self, time, prcp, temp, grad, ref_pix_hgt,
             v.long_name = 'temperature gradient'
             v[:] = grad
 
+    def write_gcm_data_file(self, time, prcp, temp, grad, ref_pix_hgt,
+                                   ref_pix_lon, ref_pix_lat):
```

Comment:
Is this enteriley new function really needed, or could we make the existing ``write_climate_data`` more flexible in order to handle GCM data, too?